newIDE/app/src/Utils/GDevelopServices/Usage.js

```diff
@@ -110,7 +110,10 @@ export type Capabilities = {|
     maxPlayersPerLobby: number,
     themeCustomizationCapabilities: 'NONE' | 'BASIC' | 'FULL',
   |},
-  versionHistory: {| enabled: boolean |},
+  versionHistory: {|
```

Comment:
Should have been

```suggestion
  versionHistory: {
```

already, there is no guarantee the backend will not return new keys (this is a good example of it happening, and it's fine!)